packages/styles/src/sass/components/button/_base.scss

```diff
@@ -0,0 +1,108 @@
+// Base Button CSS
+.cbp-btn {
+  border: 0;
```

Comment:
I think all buttons should have a border-width set so that they are sized the same and there's no jumpiness on hover/focus. Only the color changes.

Reply:
different padding is set for outline buttons vs solid and ghost to match the height in the mockups.